packages/teams-js/src/public/publicAPIs.ts

```diff
@@ -63,10 +65,12 @@ export function print(): void {
  */
 export function getContext(callback: (context: Context) => void): void {
   ensureInitializeCalled();
-  app.getContext().then((context: app.Context) => {
-    if (callback) {
-      callback(transformAppContextToLegacyContext(context));
+  sendMessageToParent('getContext', (context: Context) => {
+    if (!context.frameContext) {
```

Comment:
Thanks Erin for sharing the older V1 of getContext() call. In there it has the context.frameContext patch logics so updating here to see how it feels. With this update, the frameContext change in tests are untouched.

Reply:
(Oops, I should have looked there before looking at the v1 tests.) Glad it helped figure this out so we make the right change!